src/AsyncGenerator.yml

```diff
@@ -119,6 +119,8 @@
     - name: GetFieldValue
     - name: IsDBNull
     - name: WriteLine
+    - name: AddAction
+      containingTypeName: ActionQueue
```

Comment:
@maca88 we need a way to ignore a method if it's async counterpart is marked as obsolete. This would happen a lot in the future.

Reply:
I will [add an option](https://github.com/maca88/AsyncGenerator/issues/73) to make this possible in the following days, most likely tomorrow.

Reply:
Thanks @maca88

Reply:
[The new option](https://github.com/maca88/AsyncGenerator/commit/d9be3b7bac99920f3014469f9effa6e6c99a3352) was added and released with the version 0.7.0. To ignore all obsolete async counterparts add the following lines inside the `analyzation` property:
```
ignoreAsyncCounterparts:
- rule: Obsolete
```
where `Obsolete` is a method rule defined as:
```
- filters:
  - hasAttributeName: ObsoleteAttribute
  name: Obsolete
```